Login styling

ShinyShips opened the pull request:
This PR is for the styling of the login page, it's been made responsive and renders two different styles depending on screen size

ShinyShips commented:
![image](https://user-images.githubusercontent.com/22053630/185811051-3a788366-31bd-4970-b2fa-bf1448e4a20e.png)

ShinyShips commented:
![image](https://user-images.githubusercontent.com/22053630/185811065-9a8681e7-329d-418f-bf93-6d9e31aa43c1.png)

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -1,19 +1,30 @@
 export const Login = () => {
   const API_URL = process.env.REACT_APP_API_URL;
 
+  const [width, setWidth] = useState(window.innerWidth);
```
Remove this to line 19 in favor of using tailwind breakpoints

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
-        >
-          login
-        </button>
+  if (width > breakpoint) {
```
prefer to use tailwind breakpoints for mobile and desktop styling see here: https://tailwindcss.com/docs/responsive-design#customizing-breakpoints

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
+            />
+            <div className="w-full flex">
+              <button
+                className="w-full h-6 bg-[#3898EC] border-[#3898EC] text-white mt-2 mb-2 text-xs rounded"
```
can we add these colors to the theme

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
+            </div>
+            <div>
+              <button
+                  className="w-full h-6 bg-[#3898EC] border-[#3898EC] text-white mt-0 mb-2 text-xs rounded"
```
add colors to theme

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
+          </div>
+        </div>
+        <div className="relative w-1/2 m-0 overflow-hidden" id='image'>
+          <img className="absolute object-cover h-full" src='https://static01.nyt.com/images/2015/01/04/realestate/20150104-LIVING-slide-5SXM/20150104-LIVING-slide-5SXM-superJumbo.jpg' alt='login'/>
```
can we download this image rather than keep this in url form

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
+            />
+            <div className="w-full flex">
+              <button
+                className="w-full h-6 bg-[#3898EC] border-[#3898EC] text-white mt-2 mb-2 text-xs rounded"
```
add color to theme

alexsaltstein commented on src/components/admin/Login.js:
```diff
@@ -23,49 +34,127 @@ export const Login = () => {
+            </div>
+            <div>
+              <button
+                  className="w-full h-6 bg-[#3898EC] border-[#3898EC] text-white mt-0 mb-2 text-xs rounded"
```
add color to the theme

ShinyShips commented:
Updated to use tailwind breakpoints, Worked on general accessibility stuff using tota11y. As such, I changed the colors around just a bit to reach the 4.5 contrast ratio, removed placeholders in favor of a label inside the input, and enlarged the buttons and inputs

ShinyShips commented:
![image](https://user-images.githubusercontent.com/22053630/186193618-43da2a13-5861-4fe8-8176-cfc8684d9c7c.png)
